plan/runner.go

```diff
@@ -156,6 +158,11 @@ func (r *Runner) taskFunc(flowVal cue.Value) (cueflow.Runner, error) {
 			lg.Debug().Str("dependency", dep.Path().String()).Msg("dependency detected")
 		}
 
+		if r.dryRun {
+			lg.Info().Dur("duration", 0).Str("state", task.StateCompleted.String()).Msg(task.StateCompleted.String())
```

Comment:
Shall we add  a new state so we can show the output in a different color (blue maybe?) for dry run actions?